src/hooks/useAuthError.ts

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+
+export function useAuthError(): [React.Dispatch<string>, string] {
+  const [registrationError, setRegistrationError]: [
+    string,
+    React.Dispatch<string>,
```

Comment:
```suggestion
    React.Dispatch<React.SetStateAction<string>>,
```
It's more correct type for setState function. Please use it in all places where it's needed